roles/jail/tasks/main.yml

```diff
@@ -27,11 +52,13 @@
     mode: "0700"
   when: jail_created|changed
 
-- name: execute bootstrap script
+- name: execute bootstrap script {{ name }}
```

Comment:
Good to know. I wonder if I previously tried it with `{{ item }}` instead of a variable that was already defined.